Packs/CheckpointFirewall/ModelingRules/CheckpointFirewall_1_3/CheckpointFirewall_1_3.xif

```diff
@@ -0,0 +1,177 @@
+[MODEL: dataset ="check_point_vpn_1_firewall_1_raw"]
```

Comment:
@nkanon is the "1" should be in the dataset name?

Reply:
Yes sir, that's how it came directly from the resource. Checked in both SIEM Team and Fusion38.